Dockerfile

```diff
@@ -35,6 +35,7 @@ RUN if [ $(uname -m) = "x86_64" ]; then \
 
 FROM $BASE_IMAGE
 
+RUN dnf install -y nc
```

Comment:
the nc package should be added to the list of packages in main-packages-list.txt

Reply:
Thanks, will move it there

Reply:
done.